ansible/roles/nginx_vhost/templates/ala_cors.j2

```diff
@@ -0,0 +1,57 @@
+# Allow static fonts 
+location ~* .(eot|otf|svg|ttf|woff|woff2)$ {
+  add_header Access-Control-Allow-Origin *;
+}
+
+set $cors '';
+
+# Test your regexp of type PCRE with a tool like https://www.regextester.com/
+# Sample:  ^https?:\/\/(localhost|l-a\.site|.*\.l-a\.site)
+if ($http_origin ~* '{{ nginx_cors_origin_regexp }}') {
+  set $cors 'C';
+}
+
+# As multiple ifs are not allowed we follow this option
+# http://rosslawley.co.uk/archive/old/2010/01/04/nginx-how-to-multiple-if-statements/
+
+if ($request_method = GET) {
+  set $cors '${cors}GET';
+}
+
+if ($request_method = POST) {
+  set $cors '${cors}POST';
+}
+
+if ($request_method = OPTIONS) {
+  set $cors '${cors}OPTIONS';
+}
+
+# This is modified version of: https://enable-cors.org/server_nginx.html restricting to previous regexp
+
+if ($cors = COPTIONS) {
+  add_header 'Access-Control-Allow-Origin' '*';
```

Comment:
I think is not necessary to change the origin header here because is already done in line 10 via a regexp, so this code isi only accessed to that domains